src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/inTable.txt

```diff
@@ -65,7 +65,7 @@ Errors: Error 7-14: Invalid argument type. Cannot use Record values in this cont
 Errors: Error 12-16: Invalid argument type. Cannot use Boolean values in this context.
 
 >> {a:true} in Table({a:{b:true}})
-Errors: Error 0-8: Invalid argument type. Expecting a Record value, but of a different schema.|Error 0-8: Incompatible type. The 'a' column in the data source you’re updating expects a 'Record' type and you’re using a 'Boolean' type.|Error 12-31: Invalid argument type. Expecting a Table value, but of a different schema.|Error 12-31: Incompatible type. The 'a' column in the data source you’re updating expects a 'Boolean' type and you’re using a 'Record' type.
+Errors: Error 0-8: Invalid argument type. Expecting a Record value, but of a different schema.|Error 0-8: Incompatible type. The 'a' column in the data source you're updating expects a 'Record' type and you're using a 'Boolean' type.|Error 12-31: Invalid argument type. Expecting a Table value, but of a different schema.|Error 12-31: Incompatible type. The 'a' column in the data source you're updating expects a 'Boolean' type and you're using a 'Record' type.
```

Comment:
>u' [](http://example.com/codeflow?start=161&length=2)

Do you need to make this change, since you are normalizing the single quote on BaseRunner.cs?

Reply:
you are correct, reverted this change.